tests/modules/programs/neovim/runtime.nix

```diff
@@ -20,6 +20,13 @@ with lib;
             };
           }
         ];
+        extraWrapperArgs = let buildDeps = with pkgs; [ stdenv.cc.cc zlib ];
+        in [
+          "--suffix LIBRARY_PATH : ${lib.makeLibraryPath buildDeps}"
+          "--suffix PKG_CONFIG_PATH : ${
+            lib.makeSearchPathOutput "dev" "lib/pkgconfig" buildDeps
+          }"
+        ];
```

Comment:
you dont check this was taken into account: 
in /home/teto/hm/tests/modules/programs/neovim/./runtime.nix 's nmt.script you could check if the nvim wrapper contains `PKG_CONFIG_PATH`, aka `grep PKG_CONFIG_PATH $(which nvim)` or something like that

Reply:
I added `assertFileRegex` to nmt.script, is this OK?